### ISSUE TITLE
Authenticated server doesn't populate/show text

### ISSUE THREAD

**Issue by Vyrshkana** [EionRobb/purple-discord#331]
I'm on a number of discord servers, and while the fix to the dll fixed most of them, the one that I'm on that requires verification/phone association is now not populating the people in room panel.   Chat itself appears to be working, DMs appear to be working.

**Comment by EionRobb** [comment]
Is it a public server that I could join to see what's missing?

**Comment by Vyrshkana** [comment]
Yep!  Have [an invite](https://discord.gg/bGf8UJ3J).

**Comment by LurkerHub** [comment]
This looks to be the same as #298.
I've had a discussion with @EionRobb about it on the Pidgin discord.  I believe that the root cause is known.

**Comment by Vyrshkana** [comment]
Okay, thanks.  Weird that it worked up until I believe Thursday.

**Comment by EionRobb** [comment]
Yeah, this might be a new one related to the same protocol changes that broke the plugin from a few days ago. The list of servers and our privileges within them are broadcast differently, probably relating to the v8/v9 api changes 

I've seen it happen just now one one of the accounts I connect with, but not the other, so I think I know how to fix this one

**Comment by LurkerHub** [comment]
Will this fix also take care of #298 ?  It's a royal pain in the posterior.

**Comment by EionRobb** [comment]
So its kinda fixed by https://github.com/EionRobb/purple-discord/commit/1f103e10db3fc1c77c6f7371543172e158460bc7 but the initial load of users is busted still... but it trickle-feeds any updates.  So a bit of progress

**Comment by LurkerHub** [comment]
> the initial load of users is busted still... but it trickle-feeds any updates. So a bit of progress

Ah, that's why I am getting a barrage of notification even though I set the "large channel".

I hope you manage to nail it down.

**Comment by LurkerHub** [comment]
> So its kinda fixed by [1f103e1](https://github.com/EionRobb/purple-discord/commit/1f103e10db3fc1c77c6f7371543172e158460bc7) but the initial load of users is busted still... but it trickle-feeds any updates. So a bit of progress

I think the initial presences can be taken from:
```
{
	"t": "READY_SUPPLEMENTAL",
	"s": 2,
	"op": 0,
	"d": {
		"merged_presences": {
			"guilds": [ ... ],
			"friends": [ ... ]
		},
		"merged_members": [ ... ],
		"guilds": [ ... ]
	}
}
```
The `d/merged_presences/guilds` array is in the same order as in this structure:
```
{
	"t": "READY",
	"s": 1,
	"op": 0,
	"d": {
		"guilds": [ ... ]
	}
}
```
